### DISCUSSION TITLE
about LookAndMoveTool

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#6257] [Q&A]
I want walk the model like web play game,so find the LookAndMoveTool,but there have some problems:
1.'c' 'z' to start Collide, but just forward have colide. collide not happen when i use 'a'(to left) 、'd'(to right)、‘q' or 'e'.
2. there not have a key to jump action  ,for example , space key to jump in some play game engine.

**Comment by pmconne** [comment]
Your observations are correct. Collision detection could be improved, but it has not been a priority.
It is very unlikely we would add a "jump" feature. You could extend the tool yourself to make it more video-gamey if that is what you want.
